evidence.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package parsectpm
+
+import (
+	"crypto"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	cbor "github.com/fxamacker/cbor/v2"
+	"github.com/veraison/go-cose"
+)
+
+type Collection struct {
+	Kat *KAT `cbor:"kat" json:"kat"`
+	Pat *PAT `cbor:"pat" json:"pat"`
+}
+
+// Evidence is a wrapper around TpmParsec Attestation Collection
+type Evidence struct {
+	collection *Collection
+}
```

Comment:
why we need `Evidence` here?  It doesn't look like it's doing much.  You could just rename `Collection` to `Evidence`?